src/stability_sdk/matrix.py

```diff
@@ -0,0 +1,74 @@
+"""
+Minimal set of 4x4 column-major matrix functions for building transforms 
+compatible with the animation transform API.
+
+    [sx, 10, 20, tx]   [x]
+    [01, sy, 21, ty] . [y]
+    [02, 12, sz, tz]   [z]
+    [03, 13, 23, 33]   [1]
+
+"""
+import math
+from typing import List
+
+Matrix = List[List[float]]
```

Comment:
why are we doing this stuff with lists instead of numpy arrays? assuming the reasoning tracks, can we add the explanation/justification to the docstring for this file?

Reply:
so I wanted to implement in a way that made porting over to Typescript straightforward and so rather than use any existing Python library stuck to only basic types. on Python side of course we can use numpy and matmul but is implemented this way as explicit documentation of the matrices being column major, vectors treated as column, etc. users of the API might want to use numpy, pyglm, vispy, pytorch3d, threejs, mathjs, glmatrix, ... anything they want to build compatible transform matrix.